gtk/src/light/gtk-3.20/_common.scss

```diff
@@ -1642,6 +1642,12 @@ headerbar {
     }
   }
 
+  // GNOME 3.32+ responsive design headerbar separators
+  hdyleaflet separator.sidebar.vertical { 
```

Comment:
there's a whitespace at the end of this line

Reply:
@clobrano removed :)